package.json

```diff
@@ -41,7 +41,7 @@
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-qunit": "^4.1.1",
-    "ember-cli-release": "^1.0.0-beta.2",
+    "ember-cli-release-tag": "^0.1.5",
```

Comment:
I would recommend to change to [`release-it`](https://github.com/release-it/release-it#readme). Much more powerful and better maintained alternative.

It's often used together with [`lerna-changelog`](https://github.com/lerna/lerna-changelog), which automates the creation of a changelog based on GitHub pull requests and labels used for them. Setup could be automated with [`create-rwjblue-release-it-setup`](https://github.com/rwjblue/create-rwjblue-release-it-setup).

Reply:
I'm a believer in making releases as small as possible. For now, we let's leave this as is.